starport/pkg/cosmosgen/generate_javascript.go

```diff
@@ -212,15 +209,30 @@ func (g *jsGenerator) generateVuexModuleLoader() error {
 		return err
 	}
 
+	chainPath, err := gomodulepath.ParseAt(g.g.appPath)
+	if err != nil {
+		return err
+	}
+	chainOwner := strings.Split(chainPath.RawPath, "/")[1]
```

Comment:
We can use the following for this:

https://pkg.go.dev/github.com/tendermint/starport/starport/pkg/giturl#UserAndRepo

Reply:
We'd still have to split again or replace `/` with `-` anyway...not sure it changes much.

I based this on the module create scaffold which uses a function `owner()` on RawPath which basically does the Split as above

Reply:
solved by https://github.com/tendermint/starport/pull/987.